src/vaillant/15.basv3.tsp

```diff
@@ -0,0 +1,3286 @@
+  }
+
+  /** tariff aux heater */
+  @inherit(ri_1)
+  @ext(0x10, 0)
+  model TariffAuxHeater {
+    value: UIN;
+  }
+
+  /** adaptive heat curve */
+  @inherit(r_1, w_1)
+  @ext(0x14, 0)
+  model AdaptHeatCurve {
+    value: yesno;
+  }
+
+  /** hwc cylinder maximum charging time */
+  @inherit(r_1, w_1)
+  @ext(0x17, 0)
+  model HwcCylinderChargeMaxTime {
+    value: minutes2;
+  }
+
+  /** hwc lock time */
+  @inherit(r_1, w_1)
+  @ext(0x18, 0)
+  model HwcLockTime {
+    value: minutes2;
+  }
+
+  /** pump additional time */
+  @inherit(ri_1, wi_1)
+  @ext(0x1b, 0)
+  model PumpAdditionalTime {
+    value: minutes2;
+  }
+
+  /** hc alternative point */
+  @inherit(ri_1, wi_1)
+  @ext(0x22, 0)
+  model HcAlternativePoint {
+    @minValue(-21) @maxValue(40) @step(1)
+    value: tempv;
+  }
+
+  /** hc bivalence point */
+  @inherit(ri_1, wi_1)
+  @ext(0x23, 0)
+  model HcBivalencePoint {
+    @minValue(-20) @maxValue(30) @step(1)
+    value: tempv;
+  }
+
+  /** backup boiler */
+  @inherit(ri_1, wi_1)
+  @ext(0x24, 0)
+  model BackupBoiler {
+    value: backmode2;
+  }
+
+  /** hc emergency temperature */
+  @inherit(ri_1, wi_1)
+  @ext(0x26, 0)
+  model HcEmergencyTemperature {
+    @minValue(20) @maxValue(80) @step(1)
+    value: tempv;
+  }
+
+  /** hysteresis cylinder charge */
+  @inherit(r_1, w_1)
+  @ext(0x27, 0)
+  model HwcCylinderChargeHyst {
+    @minValue(3) @maxValue(20) @step(0.5)
+    value: calibrationv;
+  }
+
+  /** offset cylinder charge */
+  @inherit(r_1, w_1)
+  @ext(0x29, 0)
+  model HwcCylinderChargeOffset {
+    @minValue(0) @maxValue(40) @step(1)
+    value: calibrationv;
+  }
+
+  /** hwc legionella time */
+  @inherit(r_1, w_1)
+  @ext(0x2a, 0)
+  model HwcLegionellaTime {
+    value: HTM;
+  }
+
+  /** hwc legionella day */
+  @inherit(r_1, w_1)
+  @ext(0x2b, 0)
+  model HwcLegionellaDay {
+    value: daysel2;
+  }
+
+  /** next maintenance */
+  @inherit(r_1, w_1)
+  @ext(0x2c, 0)
+  model MaintenanceDate {
+    value: date;
+  }
+
+  /** offset outside temperature */
+  @inherit(r_1, w_1)
+  @ext(0x2d, 0)
+  model OffsetOutsideTemp {
+    @minValue(-3) @maxValue(3) @step(0.5)
+    value: calibrationv;
+  }
+
+  /** date */
+  @inherit(r_1)
+  @ext(0x34, 0)
+  model Date {
+    value: date;
+  }
+
+  /** time */
+  @inherit(r_1)
+  @ext(0x35, 0)
+  model Time {
+    value: HTI;
+  }
+
+  /** hydraulic scheme */
+  @inherit(ri_1, wi_1)
+  @ext(0x36, 0)
+  model HydraulicScheme {
+    value: UIN;
+  }
+
+  /** cooling outside temperature threshold */
+  @inherit(ri_1, wi_1)
+  @ext(0x38, 0)
+  model CoolingOutsideTempThreshold {
+    @minValue(10) @maxValue(30) @step(1)
+    value: tempv;
+  }
+
+  /** water pressure */
+  @inherit(r_1)
+  @ext(0x39, 0)
+  model WaterPressure {
+    value: pressv;
+  }
+
+  /** dew point offset */
+  @inherit(ri_1, wi_1)
+  @ext(0x3A, 0)
+  model DewPointOffset {
+    @minValue(-10) @maxValue(10) @step(1)
+    value: tempv;
+  }
+
+  /** total solar yield */
+  @inherit(r_1)
```

Comment:
the yield values are writable in 700.tsp as someone requested to be able to reset those when needed. maybe useful here as well (installer level I'd suggest)